sale_automatic_workflow_delivery_state/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module make `sale_automatic_workflow`'s `all_qty_delivered` field be computed using
```

Comment:
Please:
1. add to the roadmap of this module "trash it for v 14"
2. a comment in sale_automatic_workflow on top of `_compute_all_qty_delivered` that says "make it dependent on sale_delivery_state in v15 and pick the computation from ` sale_automatic_workflow_delivery_state`" or something similar

Reply:
@pedrobaeza good for you?

Reply:
OK

Reply:
Done